packages/select/src/Select/Select.spec.tsx

```diff
@@ -673,6 +673,37 @@ describe('packages/select', () => {
           expect(button).toHaveValue(optionValue);
         });
 
+        test('by space key', async () => {
+          userEvent.click(button);
+
+          const listbox = await waitFor(() => {
+            const listbox = getByRole('listbox');
+            expect(listbox).toBeVisible();
+            return listbox;
+          });
+
+          const targetOption = getByTextFor(listbox, optionText).closest('li');
+          expect(targetOption).not.toBe(null);
+
+          act(() => targetOption!.focus());
+
+          fireEvent.keyDown(targetOption!, {
+            key: keyMap.Space,
+          });
```

Comment:
Should be using `userEvent` here. This is testing our implementation of `keyMap`, not the behavior of Select

Reply:
(should change above as well)